docs/tern-lab-tutorial.md

```diff
@@ -0,0 +1,135 @@
+# Tern
+
+Tern is an inspection tool to find the metadata of the packages installed in a container image. Tern gives you a deeper understanding of your container's bill of materials so you can make better decisions about your container based infrastructure, integration and deployment strategies. It's also a good tool if you are curious about the contents of the container images you have built. This lab will show you how you can use Tern to inspect a Photon container image and also inspect the image from a resulting Dockerfile.
+
+## Environment Overview
+
+This lab tutorial assumes that you are working in a Linux environment. Your lab environment is hosted with a cloud server instance. This instance is accessible by clicking the "My Lab" icon on the left of your Strigo web page. You can access the terminal session using the built-in interface provided by Strigo or feel free to SSH directly as well (instructions will appear in the terminal).
+
+## Prerequisites
+
+### Step 1: Install environment dependencies.
+
+```
+$ sudo apt install python3 python3-pip python3-venv attr
+```
+
+### Step 2: Create a python3 virtual environment:
+
+```
+$ python3 -m venv ternenv  
+$ cd ternenv  
+$ source bin/activate  
+```
+
+## Installing Tern
+
+### Step 1: Install Tern from PyPI.
+
+```
+$ pip3 install tern
+```
+
+Tern is now ready to be used to inspect Docker container images.
+
+## Running Tern
+
+### Option 1: Generate the default report for a container image
+
+```
+$ tern -l report -o output.txt -i photon:3.0
```

Comment:
Is the environment a VM or a container image?

Reply:
@nishakm The environment is an AWS VM. There is a command line available by default for them to work on.